pkg/api/vlabs/validate.go

```diff
@@ -805,6 +808,33 @@ func (a *KubernetesConfig) Validate(k8sVersion string) error {
 	return nil
 }
 
+func (a *Properties) validateNetworkPlugin() error {
```

Comment:
Are there any values of `NetworkPlugin` that are not compatible with `NetworkPolicy` values right now other than using deprecated values of `NetworkPolicy` with a different `NetworkPlugin`? Maybe we can add a check and return an error if the user tries to set policy+plugin values that aren't compatible